src/main/scala/nl.knaw.dans.easy.solr4files/UpdateServlet.scala

```diff
@@ -34,7 +34,7 @@ class UpdateServlet(app: EasySolr4filesIndexApp) extends ScalatraServlet
 
   get("/") {
```

Comment:
I think we should remove this whole `get("/")`. It should always have been in `EasySolr4FilesServlet`